editor.planx.uk/src/@planx/components/Pay/Editor.tsx

```diff
@@ -113,6 +115,7 @@ function GovPayMetadataEditor(props: ListManagerEditorProps<GovPayMetadata>) {
             placeholder="key"
           />
           <Input
+            format={currVal.toString().startsWith("@") ? "data" : undefined}
```

Comment:
This feels like a nice solution, but it's still a "hidden" feature.

I've added copy to the section to explain this - not sure how helpful / clear / useful this is!